content/automate/ManageIQ/Transformation/Common.class/__methods__/utils.rb

```diff
@@ -0,0 +1,38 @@
+module ManageIQ
+  module Automate
+    module Transformation
+      module Common
+        class Utils
+          def self.migration_phase(handle = $evm)
+            return 'migration' if handle.root['service_template_transformation_plan_task'].present?
+            return 'cleanup' if handle.root['service_template_transformation_plan_task_id'].present?
```

Comment:
@fdupont-redhat 
the 2 values are so close the task versus the task_id and they are the same, can we add another attribute that would tell us that its cleanup.
Maybe mode=cleanup?

Reply:
Maybe test `$evm.root['object_type']` ? When in migration, it is equal to `service_template_transformation_plan_task`, which is specific, while the clean is a generic automate request.

Reply:
@fdupont-redhat I think you call the Automation Request from one of the Automate methods, I think an explicit mode=migration_cleanup might be "cleaner", all you have to do is add this extra  argument from the Automate Method when you create the request

Reply:
@mkanoor The automation request mechanism is generic and not specific to Transformation, as it is part of /System/CommonMethods/MiqAe/WeightedUpdateStatus. So setting it to `migration_cleanup` doesn't seem that good. We need key and value to be generic while explicit. Any idea ?

Reply:
As discussed, I've added a `state_machine_phase` attribute to the Transformation state machines, so that the methods can in which phase they are run.

https://github.com/ManageIQ/manageiq-content/pull/402

I will update this PR accordingly.

Reply:
@fdupont-redhat 
This PR has been merged https://github.com/ManageIQ/manageiq-content/pull/402
Can you please make the corresponding changes to this PR